src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -102,22 +103,25 @@ public String getNotImplementedString(String feature) {
    *
    * <p>For example: "LoggingServiceV2" becomes Name("Logging")
    */
-  public Name getReducedServiceName(Interface service) {
-    String name = service.getSimpleName().replaceAll("V[0-9]+$", "");
+  public Name getReducedServiceName(Interface apiInterface) {
+    String name = apiInterface.getSimpleName().replaceAll("V[0-9]+$", "");
     name = name.replaceAll("Service$", "");
     return Name.upperCamel(name);
   }
 
-  /** Human-friendly name of this service */
-  public String getServicePhraseName(Interface service) {
-    return Name.upperCamel(service.getSimpleName()).toPhrase();
+  /** Human-friendly name of this API interface */
+  public String getServicePhraseName(Interface apiInterface) {
+    return Name.upperCamel(apiInterface.getSimpleName()).toPhrase();
   }
 
   /////////////////////////////////////// Constructors /////////////////////////////////////////////
 
-  /** The name of the constructor for the service client. The client is VKit generated, not GRPC. */
-  public String getApiWrapperClassConstructorName(Interface interfaze) {
-    return publicClassName(Name.upperCamel(interfaze.getSimpleName(), "Client"));
+  /**
+   * The name of the constructor for the apiInterface client. The client is VKit generated, not
```

Comment:
Unrelated to this change, but the second sentence of this comment should be removed.